squad/frontend/templates/squad/build-nav.html

```diff
@@ -18,11 +18,17 @@ <h1>
             All test results
         </a>
     </li>
+    {% if build.test_jobs.all %}
+    {% load filter_jobs %}
     <li role="presentation" {% if url_name == 'testjobs' %}class="active"{% endif %}>
         <a href="{% build_section_url build 'testjobs' %}">
             Test jobs
         </a>
+        {% for i in build|filter_jobs %}
+        <span class="badge alert-success" data-toggle="tooltip" data-placement="top" title="Pass">{{ i.0 }} {{ i.1 }}</span>
```

Comment:
won't this make the navigation tab too wide? say we had at least 3 different job status ...

Reply:
Yeah, I know. The UI might be difficult to do given I don't want to hardcode the names of the job statuses in the templates. I'll try to find some solution. This part is WIP.